stream_alert_cli/terraform/generate.py

```diff
@@ -0,0 +1,400 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from fnmatch import fnmatch
+import json
+import os
+import string
+
+from stream_alert.shared.metrics import FUNC_PREFIXES
+from stream_alert_cli.logger import LOGGER_CLI
+from stream_alert_cli.terraform._common import (
+    DEFAULT_SNS_MONITORING_TOPIC,
```

Comment:
Ha, it is here. If you import `DEFAULT_SNS_MONITORING_TOPIC` here, please remove line 42, where redefine it.

Reply:
@chunyong-lin I pushed the change